### PR TITLE
Workshop notebook

### PR DESCRIPTION
**Pull request by cdolfi**

### CODE REVIEW

File: notebooks/issue_first_resposne.ipynb [hunk 1]
```diff
@@ -0,0 +1,11971 @@
+{
```
Comment (JamesKunstle) [inline notebooks/issue_first_resposne.ipynb:1]: <p>rename notebook - "issues_first_resposne"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oss-aspen/Rappel/pull/250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (JamesKunstle) [inline notebooks/issue_first_resposne.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #9.    #from jupyter_dash import JupyterDash
</pre><p>remove unused imports</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oss-aspen/Rappel/pull/250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (JamesKunstle) [inline notebooks/issue_first_resposne.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #17.    with open("../copy_cage-padres.json") as config_file:
</pre><p>rename config - maybe "augur_config.json" for sake of decreasing ambguity.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oss-aspen/Rappel/pull/250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (JamesKunstle) [inline notebooks/issue_first_resposne.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.    df["closed_at"] = pd.to_datetime(df["closed_at"], utc=True)
</pre><p>recently read the pandas docs - when this executes, you can pass a list of columns to read in as datetime with the <code><strong>parse_dates</strong></code><strong> </strong>argument, so you don't have to do these steps separately.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oss-aspen/Rappel/pull/250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (JamesKunstle) [inline notebooks/issue_first_resposne.ipynb:1]: <p>Title graph</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oss-aspen/Rappel/pull/250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (cdolfi) [reply notebooks/issue_first_resposne.ipynb:1]: Can you open an issue on 8knot describing this? I'd like to keep this the same for the workshop and make a codebase wide change

Comment (JamesKunstle) [reply notebooks/issue_first_resposne.ipynb:1]: <p>Bug</p>

**Comment by JamesKunstle** [comment]
@cdolfi The only notebook in the PR now is plotly_tutorial.ipynb

**Review by JamesKunstle — APPROVED** [review]

**Pull request merged**